src/libdredd/include/libdredd/mutation_replace_binary_operator.h

```diff
@@ -59,6 +61,13 @@ class MutationReplaceBinaryOperator : public Mutation {
   [[nodiscard]] bool IsValidReplacementOperator(
       clang::BinaryOperatorKind op) const;
 
+  // This returns a string corresponding to the non-mutated expression.
+  std::string getExpr(clang::ASTContext& ast_context) const;
```

Comment:
The name should be `GetExpr` - see https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Done.